tests/linalg_test.py

```diff
@@ -1131,5 +1131,31 @@ def testIssue2131(self, n, dtype):
                             check_dtypes=True)
     self._CompileAndCheck(jsp_fun, args_maker_zeros, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name":
+       "_lhs={}_rhs={}".format(
+           jtu.format_shape_dtype_string(lhs_shape, dtype),
+           jtu.format_shape_dtype_string(rhs_shape, dtype)),
+       "lhs_shape": lhs_shape,
+       "rhs_shape": rhs_shape, "dtype": dtype, "rng_factory": rng_factory}
+      for lhs_shape, rhs_shape in [
+          ((2, 4), (2, 2)),
+          ((5, 4), (5, 2)),
+          ((4, 4), (4, 1))
```

Comment:
It would be good to add the test case with `rhs_shape = (i, )` (it is different from `(i, 1)`).